pvlib/iotools/acis.py

```diff
@@ -0,0 +1,483 @@
+
+    # somewhat inconveniently, the ACIS API tends to return errors as "valid"
+    # responses instead of using proper HTTP error codes:
+    if "error" in payload:
+        raise requests.HTTPError(payload['error'], response=response)
+
+    columns = ['date'] + [e['name'] for e in params['elems']]
+    df = pd.DataFrame(payload['data'], columns=columns)
+    df = df.set_index('date')
+    df.index = pd.to_datetime(df.index)
+    df.index.name = None
+
+    metadata = payload['meta']
+    # for StnData endpoint, unpack combination "ll" into lat, lon 
+    ll = metadata.pop('ll', None)
+    if ll:
+        metadata['lon'], metadata['lat'] = ll
+
+    try:
+        metadata['elev'] = metadata['elev'] * 0.3048  # feet to meters
+    except KeyError:
+        # some queries don't return elevation
+        pass
+
+    if map_variables:
+        df = df.rename(columns=VARIABLE_MAP)
+
+        for key in list(metadata.keys()):
+            if key in VARIABLE_MAP:
+                metadata[VARIABLE_MAP[key]] = metadata.pop(key)
+
+    return df, metadata
+
+
+def get_acis_prism(latitude, longitude, start, end, map_variables=True,
+                   url="https://data.rcc-acis.org/GridData", **kwargs):
+    """
+    Retrieve estimated daily precipitation and temperature data from PRISM
+    via the Applied Climate Information System (ACIS).
+
+    Geographical coverage: approximately -130° to -65° in longitude and
+    0° to 50° in latitude.
+
+    Parameters
+    ----------
+    latitude : float
+        in decimal degrees, between -90 and 90, north is positive
+    longitude : float
+        in decimal degrees, between -180 and 180, east is positive
+    start : datetime-like
+        First day of the requested period
+    end : datetime-like
+        Last day of the requested period
+    map_variables : bool, default True
+        When True, rename data columns and metadata keys to pvlib variable
+        names where applicable. See variable :const:`VARIABLE_MAP`.
+    url : str, default: 'https://data.rcc-acis.org/GridData'
+        API endpoint URL
+    kwargs:
+        Optional parameters passed to ``requests.get``.
+
+    Returns
+    -------
+    data : pandas.DataFrame
+        Daily precipitation [mm] and temperature [Celsius] data
+    metadata : dict
+        Metadata of the selected grid cell
+
+    Raises
+    ------
+    requests.HTTPError
+        A message from the ACIS server if the request is rejected
+
+    Notes
+    -----
+    PRISM data is aggregated from 12:00 to 12:00 UTC, meaning data labeled
+    May 26 reflects to the 24 hours ending at 7:00am Eastern Standard Time
+    on May 26.
+
+    References
+    ----------
+    .. [1] `PRISM <https://prism.oregonstate.edu/>`_
+    .. [2] `ACIS Gridded Data <http://www.rcc-acis.org/docs_gridded.html>`_
+    .. [3] `ACIS Web Services <http://www.rcc-acis.org/docs_webservices.html>`_
+
+    Examples
+    --------
+    >>> df, meta = get_acis_prism(40, -80, '2020-01-01', '2020-12-31')
```

Comment:
```suggestion
    >>> df, meta = pvlib.iotools.get_acis_prism(
    >>>     latitude=40, longitude=-80, start='2020-01-01', end='2020-12-31')
```
The users who find examples useful might easily not be aware that the function is in the iotools module. In any case, I think it's nice that it works out of the box without special imports.